packages/terra-form-select/src/_AddOption.jsx

```diff
@@ -1,35 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
 import Option from './_Option';
 
 const propTypes = {
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.